### DISCUSSION TITLE
List pull requests api doesn't work for github enterprise

### DISCUSSION THREAD

**Discussion by FlyBooks** [community/community#24329] [Enterprise]
<p>jobs:<br>
test:<br>
runs-on: broswer<br>
steps:<br>
-put link <a href="https://api.github.com/repos/imdev/gmp/pulls/comments" rel="noopener nofollow ugc">https://api.github.com/repos/imdev/gmp/pulls/comments</a> in google chrome</p>
<p>but it doesn’t work, because our github is enterprise and domain is not <a href="http://github.com" rel="noopener nofollow ugc">github.com</a></p>

**Comment by canuckjacq** [comment, marked as answer]
<p>:wave: Welcome!</p>
<p>You will have to ask your GitHub Enterprise admin what the correct path is for your domain.</p>

**Comment by FlyBooks** [comment]
<p>thank you very much!</p>
